src/main/groovy/wooga/gradle/snyk/cli/commands/TestProjectCommandSpec.groovy

```diff
@@ -348,6 +492,10 @@ trait SnykTestArgumentSpec extends BaseSpec implements ContextSpecificArgumentSp
         policyPath.set(value)
     }
 
+    void setPolicyPath(String value) {
+        policyPath.set(new File(value))
+    }
+
```

Comment:
I need to be reminded why we needed the string overload?
Because from the gradle script one can easily create a file from string with `file()`

Reply:
I think it was because I wanted to test all the file properties with strings but I ended up not doing that.

Reply:
So we actually don't need it then?